### PR TITLE
Advanced > OData: remove valuelist from snippet

### PR DESCRIPTION
**Pull request by renejeglinsky**

### CODE REVIEW

File: advanced/odata.md [hunk 1]
```diff
@@ -216,10 +216,6 @@ OData defines a strict two-fold key structure composed of `@<Vocabulary>.<Term>`
 
 ```cds
 @Common.Label: 'Customer'
-@Common.ValueList: {
```
Comment (renejeglinsky) [inline advanced/odata.md:219]: Or should this be replaced by sth else?

Comment (ralfhandl) [reply advanced/odata.md:219]: Do you want
1. a structured annotation on an entity, or 
2. a correct ValueList example?

If 1, then use @UI.HeaderInfo, see for example https://pages.github.tools.sap/cap/docs/guides/extensibility/customization#extending-ui-annotations

**Comment by stewsk** [comment]
Hi @Rene, what is the purpose of this chang?

**Comment by renejeglinsky** [comment]
> Hi @rene, what is the purpose of this chang?

I sent you an email.

**Comment by stewsk** [comment]
I think we want an example for a structured annotation at that place.
Have added an example with `@UI.Header` as proposed by @ralfhandl

**Review by ralfhandl — APPROVED** [review]

**Pull request merged**